igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/JenkinsConfig.groovy

```diff
@@ -52,11 +58,14 @@ class JenkinsConfig {
         return new JenkinsService(jenkinsHostId, jenkinsClient)
     }
 
-    static JenkinsClient jenkinsClient(String address, String username, String password) {
+    static JenkinsClient jenkinsClient(String address, String username, String password, int timeout = 30) {
+        OkHttpClient client = new OkHttpClient()
+        client.setReadTimeout(timeout, TimeUnit.SECONDS)
```

Comment:
If we're using okHttpClient in here ... feasibility on using the `OkHttpClientConfiguration` we use elsewhere, it would result in similar configuration patterns across all the services.

```
okHttpClient:
  readTimeoutMs: 59000
  connectTimoutMs: 30000
```